CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+# Contributing to Panda3D
+Thank you for looking into ways to contribute to Panda3D.
+Most information on how to contribute and get involved already exists in other places, so this page serves as another way to find those resources.
```

Comment:
The "so" in this line is really starting to bother me, I'm going to reword this.